packaging/nix/umu-launcher.nix

```diff
@@ -1,4 +1,4 @@
-{lib, pyth1 ,python3Packages , umu-launcher, pkgs,version, truststore ? true, ...}:
+{lib, pyth1 ,python3Packages , umu-launcher, pkgs,version, truststore ? true, cbor2 ? true, xxhash ? true, rustPlatform, ...}:
```

Comment:
nitpick: cbor2 & xxhash might be better represented as `deltaUpdates ? true`

Reply:
Added `deltaUpdates ? true`